facp.md

```diff
@@ -0,0 +1,82 @@
+# FACP
+
+The FADT, of which the signiture is FACP, refers to the Fixed ACPI Description Table, and is pointed by the RSDT.
```

Comment:
Ok now it is better, but i think you should add more info, imagine me (actually you don't have to imagine this is the truth) i have arrived here because i wanted to implement facp, and start to read these notes, at the end of them i still have this question: "Ok but what is their purpose? Why i should implement them". 

Try to explain what is FACP doing and when we should use it.